docs/manage/partitions/flex/estimate-scan-data.md

```diff
@@ -47,6 +47,22 @@ To view the Daily scan and Yearly scan data information based on a log-based SLO
 
 <img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/scan-estimates-slo.png')} alt="scan-estimates-slo" style={{border:'1px solid gray'}} width="600" />
 
+
+## Dashboard scan estimates
+
+When you enter a query to create a dashboard, Sumo Logic will estimate and display the amount of data scanned in Flex. This detail is vital as with Flex pricing, you're charged for the amount of data that's scanned to complete the query. 
```

Comment:
```suggestion
When you enter a query to create a dashboard, Sumo Logic will estimate and display the amount of data scanned in Flex. This detail is crucial because, with Flex Pricing, you're charged based on the amount of data scanned to complete the query.
```